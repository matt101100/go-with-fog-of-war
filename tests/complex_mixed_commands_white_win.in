who
view
place J10
place J11
place K10
view
place K11
history
place K11
who
place K12
place L14
view
place L13
place I12
place I10
view
place J9
place M3
who
history
view
place A1
resign
