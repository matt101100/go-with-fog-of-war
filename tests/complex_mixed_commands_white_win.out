B
J10,.................................................
P18,xxxxxxxxxxxxxx...................................
J10J11K10K11
Occupied coordinate
B
A11,xxx....xxx....xxx....xxx....xxx....xxx....xxx....
S18,xxxxxxxxxxxxxx....xxx....xxx....xxx....xxx....xxx
W
J10J11K10K11K12L14L13I12I10J9M3
O1,........#...................xxxxxxxxxxxxxxxxxxxxx
White wins!
J10J11K10K11K12L14L13I12I10J9M3A1
Thank you for playing!
